spec/requests/users_controller_spec.rb

```diff
@@ -5899,6 +5900,194 @@ def create_totp
     end
   end
 
+  describe "#create_passkey" do
+    before { SiteSetting.experimental_passkeys = true }
+
+    it "fails if user is not logged in" do
+      stub_secure_session_confirmed
+      post "/u/create_passkey.json"
+
+      expect(response.status).to eq(403)
+    end
+
+    it "stores the challenge in the session and returns challenge data, user id, and supported algorithms" do
+      sign_in(user1)
+      stub_secure_session_confirmed
+      post "/u/create_passkey.json"
+
+      secure_session = read_secure_session
+      response_parsed = response.parsed_body
+      expect(response_parsed["challenge"]).to eq(DiscourseWebauthn.challenge(user1, secure_session))
+      expect(response_parsed["rp_id"]).to eq(DiscourseWebauthn.rp_id)
+      expect(response_parsed["rp_name"]).to eq(DiscourseWebauthn.rp_name)
+      expect(response_parsed["user_secure_id"]).to eq(
+        user1.reload.create_or_fetch_secure_identifier,
```

Comment:
```suggestion
        user1.create_or_fetch_secure_identifier,
```

Small one but I don't think the reload is necessary.